frontend/src/components/AllTripsOverviewCard.tsx

```diff
@@ -0,0 +1,13 @@
+import {Trip} from "../model/Trip";
+
+type AllTripsOverviewCardProps = {
+    trips: Trip[]
+}
+
+export default function AllTripsOverviewCard({trips}:AllTripsOverviewCardProps) {
+
+    return (
+    <div className={"all-time-total"}>
+
+    </div>)
```

Comment:
This code might be removed?

Reply:
Yes, for this PR it is not necessary. It is a new project for today.